src/clib/lib/include/ert/job_queue/job_queue_status.hpp

```diff
@@ -22,8 +22,6 @@ struct job_queue_status_struct {
         JOB_QUEUE_IS_KILLED, // Job has been killed, due to JOB_QUEUE_DO_KILL, FINAL STATE - controlled by job_queue
         JOB_QUEUE_DO_KILL, // User / queue system has requested killing of job - controlled by job_queue / external scope
         JOB_QUEUE_SUCCESS, // All good, comes after JOB_QUEUE_DONE, with additional checks, FINAL STATE - controlled by job_queue
-        JOB_QUEUE_RUNNING_DONE_CALLBACK, // Temporary state, while running requested callbacks after an ended job - controlled by job_queue
-        JOB_QUEUE_RUNNING_EXIT_CALLBACK, // Temporary state, while running requested callbacks after an ended job - controlled by job_queue
```

Comment:
This array is tied to size `#define JOB_QUEUE_MAX_STATE 16` which is not the case anymore.
Best case is that the missing two elements now have value zero.

The affected code should work fine still, but this isn't great.

Reply:
This is the 2^n I mentioned:

```
job_queue_status_get_count(
        job_status,
        pow(2,
            JOB_QUEUE_MAX_STATE)); // This enum value is completly missing; should give util_abort.
```